app/controllers/mixins/menu_update_mixin.rb

```diff
@@ -0,0 +1,11 @@
+module Mixins
+  module MenuUpdateMixin
+    def menu_section_url
+      url = CGI.unescape(params[:url])
```

Comment:
should this be `URI.unescape`?

(not sure, we have both, but only the documentation for URI mentions whole urls)

Reply:
Codecliemate & rubocop would complain: `URI.unescape` method is obsolete and should not be used. Instead, use `CGI.unescape`, ...

Reply:
Oh, :+1: then.